benchmarks/encoder/scenario.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import pyperf
+from util import gen_traces
+
+from ddtrace.internal.encoding import Encoder
+
+
+try:
```

Comment:
Ah probably worth having a comment here that the encoder changed and so this is here to support comparing across versions for the near future.

Reply:
```suggestion
try:
    # the introduction of the buffered encoder changed the internal api
    # see https://github.com/DataDog/dd-trace-py/pull/2422
```